product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/06_metrics.mdx

```diff
@@ -86,44 +86,43 @@ removed or renamed. Where feasible, an effort will be made not to change the
 meaning or type of existing metrics without also changing the metric name.
 
 At time of writing, all metrics forwarded from Prometheus are in the
-`prometheus` namespace. This might change in a future release.
+`DpMetrics_CL` table. This might change in a future release.
 
 Effective use of the available metrics requires an understanding of Azure
 time-series data, metrics dimensions, and of the tagging conventions used in
 the metrics streams.
 
-### Metrics tags
+### Metrics labels
 
-All Postgres metrics share a common tagging scheme. Entries generally
-have at least the following tags:
+All Postgres metrics share a common labelling scheme. Entries generally
+have at least the following labels:
 
 | Name                         | Description                                                   |
 | ---------------------------- | ------------------------------------------------------------- |
-| address                      | IP address of the host the metric originated from             |
+| instance                     | IP address of the host the metric originated from             |
 | postgresql                   | BigAnimal postgres cluster identifier, e.g., `p-abcdef012345` |
 | role                         | Postgres instance role, "primary" or "replica"                |
 | datname                      | Postgres database name (where applicable)                     |
-| pod_name                     | k8s pod name                                                  |
-| hostName                     | AKS node host name                                            |
-| container.azm.ms/clusterName | AKS cluster name                                              |
+| pod                          | k8s pod name                                                  |
+| aks_cluster_name             | AKS cluster name                                              |
 
-When querying for tags, for best performance apply filters that don't
-require inspection of tags (for example, filters by metric name) before tag-based filters.
+When querying for labels, for best performance apply filters that don't
+require inspection of labels (for example, filters by metric name) before label-based filters.
```

Comment:
This reads awkwardly - I understand the concept of filtering before examining the values, and I appreciate that this is the original wording with the terminology updated.  I am unsure if this adds value to the customer perspective.  Approving as this is the existing published phrasing.